chimney/src/main/scala/io/scalaland/chimney/dsl/TransformerInto.scala

```diff
@@ -20,6 +21,9 @@ final class TransformerInto[From, To, C <: TransformerCfg](
     val td: TransformerDefinition[From, To, C]
 ) {
 
+  def lift[F[_]]: TransformerFInto[F, From, To, WrapperType[F, C]] =
+    new TransformerFInto[F, From, To, WrapperType[F, C]](source, td.lift[F])
```

Comment:
Why no implicit TransformerFSupport[F] here and in another places where F[_] appears?

Reply:
Implicit instance of `F[_]` is resolved by a macro that expands transformer (where it's really needed in order to construct transformer tree). We could add the instance here, but it would be not used. We couldn't capture it on the type-level configuration (as the resolved instance is a value-level thing).

Theoretically we could carry it on the value-level (as a member of `TransformerFDefinition`), but such member would have to be public and it wouldn't be worth the effort.